RELEASE_NOTES.md

```diff
@@ -8,6 +8,10 @@ This document contains release notes for the current major release and all patch
 For prior releases,
 see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 12.17.0 (2017-10-10)
+
+This release updates the version of java to 8u144 to address CVE-2017-3526.
```

Comment:
I might be misreading the CHANGELOG (again!) -- but it looks like the stats endpoint has gone unmentioned so far: [see here](https://github.com/chef/chef-server/blob/master/CHANGELOG.md#merged-pull-requests-1)

Reply:
@srenatus Good catch!  We DEFINITELY want to talk about the stats feature in the release notes.  It is great.

Reply:
👍